src/config.ts

```diff
@@ -49,81 +49,6 @@ export const config: BotConfig = {
 		["Sea Serpent", "#5db9cf"],
 	],
 
-	// --- Activity Tracker ---
-	logActivity: true,
```

Comment:
A way to shut of the feature completely is necessary. Please bring it back. It was also really easy to implement

Reply:
Sure